### ISSUE TITLE
grammar3.7.txt please

### ISSUE THREAD

**Issue by themanwithnoplan** [brennv/autocomplete-python-jedi#24]
no autocomplete couldnt find grammar3.7.txt so i switched name on grammar3.6.txt to *3.7.txt, it works now.
When will there be a new one from yous?

**Comment by sumagnadas** [comment]
Yes please make a grammar3.7.txt file because that's the latest version in ubuntu 19.10 and i also did what @themanwithnoplan did and it works for now. So please try to make an update for the package and do it fast.

**Comment by mijorus** [comment]
thanks
